notifications_utils/version.py

```diff
@@ -1 +1 @@
-__version__ = '10.0.0'
+__version__ = '10.0.1'
```

Comment:
minor thing (heh) - should probably be 10.1.0 as it's new/changed functionality rather than bug fixes/refactors.

actually, possibly it should be 11.0.0 as it's a breaking change to downstream integrators (all our statsd graphs will need to be rewritten?)